addons/mail/static/src/discuss/core/common/notification_settings.scss

```diff
@@ -0,0 +1,7 @@
+.o-discuss-NotificationSettings {
+    width: 150px;
+
+    .o-discuss-NotificationSettings-muteText {
+        font-size: smaller;
+    }
```

Comment:
I would put a `o-smaller` classname in `@mail/core.scss`, so that the footprint in template and SCSS is smaller: